server/src/main/java/org/elasticsearch/action/support/master/AcknowledgedRequest.java

```diff
@@ -41,6 +41,11 @@
     protected AcknowledgedRequest() {
     }
 
+    protected AcknowledgedRequest(StreamInput in) throws IOException {
```

Comment:
This change is not really related to the unfollow api, but it allows this: https://github.com/elastic/elasticsearch/pull/34132/files#diff-bd430812001c493a52293cc54e56ea24R36 (immutable fields)

and I think it good that at some point we can not use `Streamable` anymore. I want to go over other ccr request classes in a follow up change.